CDP4JsonFileDal.Tests/JsonFileDalTestFixture.cs

```diff
@@ -99,12 +109,23 @@ public void OneTimeSetUp()
         public void SetUp()
         {
             var path = Path.Combine(TestContext.CurrentContext.TestDirectory, "files", "LOFT_ECSS-E-TM-10-25_AnnexC.zip");
+            var migrationSourceFile = Path.Combine(TestContext.CurrentContext.TestDirectory, @"..\..\..\files", "migration.json");
+
+            this.annexC3File = Path.Combine(TestContext.CurrentContext.TestDirectory, "files", "AnnexC3.zip");
+            this.migrationFile = Path.Combine(TestContext.CurrentContext.TestDirectory, "files", "migration.json");
+
+            if (!File.Exists(this.migrationFile))

```

Comment:
Does this try to copy the test migration file from outside the project, if it does not exist?

Reply:
See above comment.